<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EControlSysClass</class>
 <widget class="QMainWindow" name="EControlSysClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1343</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>空军勤务保障模拟训练系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
background-color: rgb(70, 140, 232);
}
QToolButton{
 	font-family:Microsoft Yahei;
    font-size:13pt;
	color: rgb(255, 255, 255);
}
.QPushButton:hover,.QToolButton:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #F2F9FF,stop:1 #DAEFFF);
}
.QPushButton:pressed,.QToolButton:pressed{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DEF0FE,stop:1 #C0DEF6);
}
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>75</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/EControlSys/res/u17.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; color:#ffffff;&quot;&gt;空军勤务保障模拟训练系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="plan">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>想定</string>
         </property>
         <property name="icon">
          <iconset resource="EControlSys.qrc">
           <normaloff>:/EControlSys/res/plan.png</normaloff>:/EControlSys/res/plan.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>64</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="control">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>导调</string>
         </property>
         <property name="icon">
          <iconset resource="EControlSys.qrc">
           <normaloff>:/EControlSys/res/control.png</normaloff>:/EControlSys/res/control.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>64</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="kaoshi">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>考试管理</string>
         </property>
         <property name="icon">
          <iconset resource="EControlSys.qrc">
           <normaloff>:/EControlSys/res/kaoshi.png</normaloff>:/EControlSys/res/kaoshi.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>64</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="system">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>系统管理</string>
         </property>
         <property name="icon">
          <iconset resource="EControlSys.qrc">
           <normaloff>:/EControlSys/res/system.png</normaloff>:/EControlSys/res/system.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>64</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>序号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>终端</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>操作ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>说明</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionqidong">
   <property name="text">
    <string>qidong</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EControlSys.qrc"/>
 </resources>
 <connections/>
</ui>
